### DISCUSSION TITLE
No projects visible on the cloud

### DISCUSSION THREAD

**Discussion by TheJoseph24** [opengisch/QField#3184] [Bugs]
Hi!
I´m doing a lot of fieldwork and I have (or had) several projects on my account, as well my coworkers (shared project).
Today, after a hard working morning, we came home, synced our projects through our apps. However, when I went to give the sync on my PC, the process was taking way too long, so I decided to check out the Qfield website, and for my surprise, none of my projects were there, They all litterally dissapeared. We inittially tought it could be a server error, but it seems less and less like one. 
Someone help, please!

**Comment by suricactus** [comment]
Follow the thread here: https://github.com/opengisch/QField/discussions/3181
